var/spack/repos/builtin/packages/rdma-core/libdrm.patch

```diff
@@ -0,0 +1,10 @@
+--- CMakeLists.txt.ori	2024-06-24 15:48:01.389773228 +0200
++++ CMakeLists.txt	2024-06-24 15:48:42.126485322 +0200
+@@ -540,6 +540,7 @@
+ 
+ # Check if the headers have been installed by kernel-headers
+ find_path(DRM_INCLUDE_DIRS "drm.h" PATH_SUFFIXES "drm")
++find_path(DRM_INCLUDE_DIRS "drm.h" PATH_SUFFIXES "libdrm")
```

Comment:
This will not work on other platforms that need "drm" instead. As `PATH_SUFFIXES` accepts a list, it's enough to *add* `"libdrm"` and now `find_path` will try both if necessary and thus always succeed on any platform where it used to work, as well as on any that use `libdrm` (such as ArchLinux).

Reply:
See https://cmake.org/cmake/help/latest/command/find_path.html

Reply:
```suggestion
+find_path(DRM_INCLUDE_DIRS "drm.h" PATH_SUFFIXES "drm" "libdrm")
```

Reply:
You are right; my upstream package follows your logic. Somehow, to avoid confusion, I thought it would be clear to split it into a new line, but maybe this isn't equivalent. Nevertheless, I totally agree with your change as it is identical to mine upstream.

https://github.com/linux-rdma/rdma-core/pull/1477